extensions/quick-access/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quick-access",
+  "title": "Quick Access",
+  "description": "Quick copy, open, reveal the latest files in pinned Finder directories.",
+  "icon": "extension-icon.png",
+  "author": "koinzhang",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "pin-directory",
+      "title": "Pin Directory",
+      "icon": "pin-directory.png",
+      "subtitle": "Quick Access",
+      "description": "Pin directories that can be accessed quickly. To use this command, you need to select folders or open a Finder window first.",
+      "mode": "no-view"
+    },
+    {
+      "name": "search-pinned-directory",
+      "title": "Search Pinned Directory",
+      "icon": "access-directory.png",
+      "subtitle": "Quick Access",
+      "description": "Quick copy, open, reveal the latest files in pinned Finder directories.",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "autoCopyLatestFile",
+          "type": "checkbox",
+          "required": true,
```

Comment:
Let's not mark this as required and pick the default that you think is best. I think not copying it automatically. Mostly because I see this screen when I open the extension for the first time and I don't know what to do here.
```suggestion
          "required": false,
```
![CleanShot 2022-04-22 at 4 04 15](https://user-images.githubusercontent.com/12066405/164741799-f44a3221-febd-4163-a4c8-6e37bdd1a63a.png)

Reply:
Okay, I'll set it to false by default and stop popping up.